app/include/dt-bindings/zmk/keys.h

```diff
@@ -6,6 +6,8 @@
 
 #pragma once
 
+#define EXT_USAGE(page, id) ((page << 16) + id)
```

Comment:
#153 used the bitwise `OR` notation.  We should be consistent whichever we settle on.